backend/app/Http/Middleware/MustBeAdministrator.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
rename to Administrator since it is understandabl. please avoid "MustBe" in filenames